src/manage_nsfs/nc_lifecycle.js

```diff
@@ -807,21 +812,23 @@ class NCLifecycle {
      * } params
      * @returns {Void}
     */
-    update_status({ bucket_name, rule_id, op_name, op_times, reply = [], error = undefined }) {
+    update_status({ mount_point, bucket_name, rule_id, op_name, op_times, reply = [], error = undefined }) {
```

Comment:
not for this PR. but I think this function is getting messier with time. I think we should consider better Polymorphism. maybe even use objects?